Dockerfile.monitor

```diff
@@ -18,7 +18,8 @@ FROM registry.access.redhat.com/ubi9
 COPY --from=builder /workdir/src/runtime/kata-monitor /usr/bin/kata-monitor
 
 # Red Hat labels
-LABEL name="openshift-sandboxed-containers-operator-monitor" \
+LABEL name="openshift-sandboxed-containers/osc-monitor-rhel9" \
+cpe="cpe:/a:redhat:confidential_compute_attestation:1.10::el9" \
 version="1.10.2" \
```

Comment:
Maybe not part of this PR but shouldn't we have 1.10.3 now on osc-release-v1.10 ?

Reply:
Done !